schemas/JSON/settings/settings.schema.0.2.json

```diff
@@ -91,12 +91,24 @@
         }
       }
     },
+    "InstallRoot": {
+      "description": "The root directory where packages are installed to. Applies to the portable installer type.",
```

Comment:
We need a blanket term that applies to all packages that WPM is taking ownership of the install action.  No idea what it is.